packages/now-build-utils/test/unit.test.ts

```diff
@@ -0,0 +1,113 @@
+import assert from 'assert';
```

Comment:
I just realized that this will make two unit tests files: `unit.test.js` and `unit.test.ts`. This seems very confusing. Can you instead name this one something like `unit.test.detectors.ts` so we don't think one generates the other?